src/api/feedback/feedback.ts

```diff
@@ -0,0 +1,30 @@
+import api, { ApiRequestConfig, ApiTargets } from '@api';
+import { QueryFunction, useQuery } from '@tanstack/react-query';
+import { IFeedbackParams, FeedbackADSQuery, IADSApiFeedbackResponse } from './types';
+
+const feedbackKeys = {
+  general: (params: IFeedbackParams) => ['feedback/general', { params }] as const,
+};
+
+export const useFeedback: FeedbackADSQuery = (params, options) => {
+  return useQuery({
```

Comment:
I've found that useMutation is better choice for these types of queries.  I'm thinking we'll start to transition over to using that for any non-GET requests in the future